src/wp-includes/formatting.php

```diff
@@ -5394,6 +5394,28 @@ function normalize_whitespace( $str ) {
  * @return string The processed string.
  */
 function wp_strip_all_tags( $text, $remove_breaks = false ) {
+	if ( is_null( $text ) ) {
+		return '';
+	}
+
+	if ( ! is_string( $text ) ) {
```

Comment:
```suggestion
	if ( ! is_scalar( $text ) ) {
```

https://3v4l.org/gRSnZ shows that a string check is stricter than PHP 8, which handles the implicit conversion of scalar values.

Reply:
[9862146](https://github.com/WordPress/wordpress-develop/pull/3132/commits/986214639da8dff312db5ade06e5a79e64f8cac6)